packages/cozy-client/src/__snapshots__/CozyClient.spec.js.snap

```diff
@@ -2,10 +2,6 @@
 
 exports[`CozyClient queryAll should call \`query\` until there is no more document to query 1`] = `
 Array [
-  Object {
-    "_id": "0",
-    "label": "Shopping",
-  },
```

Comment:
if this test changes, this is a breaking change. And I don't know if we want a BC here. Why this snapshot has changed?

Reply:
Je pense que le fait de ne pas avoir la première entrée du tableau avant n'était pas normal... ?

Reply:
Il était présent, là tu le supprimes

Reply:
Ah mince exact ! J'ai lu à l'envers 🤦‍♂️

Reply:
c'est corrigé, j'ai dû ajouter un `document.push` avant la boucle : https://github.com/cozy/cozy-client/blob/fix/queryAllJF/packages/cozy-client/src/CozyClient.js#L926